server/command.go

```diff
@@ -126,28 +141,50 @@ func (p *Plugin) settingsError(userID string, channelID string, errorText string
 }
 
 func (p *Plugin) executeSettingsCommand(c *plugin.Context, args *model.CommandArgs, parameters []string) (*model.CommandResponse, *model.AppError) {
+	l := p.getUserLocalizer(args.UserId)
 	text := ""
 
 	userConfig, err := p.getUserConfig(args.UserId)
 	if err != nil {
 		mlog.Debug("Unable to get user config", mlog.Err(err))
-		return p.settingsError(args.UserId, args.ChannelId, "Unable to get user settings.")
+		return p.settingsError(args.UserId, args.ChannelId, l.MustLocalize(&i18n.LocalizeConfig{
+			DefaultMessage: &i18n.Message{
+				ID:    "jitsi.command.settings.unable_to_get",
+				Other: "Unable to get user settings",
+			},
+		}))
 	}
 
 	if len(parameters) == 0 {
-		text = fmt.Sprintf("###### Jitsi Settings:\n* Embedded: `%v`\n* Naming Scheme: `%s`", userConfig.Embedded, userConfig.NamingScheme)
+		text = l.MustLocalize(&i18n.LocalizeConfig{
+			DefaultMessage: &i18n.Message{
+				ID: "jitsi.command.settings.current_values",
+				Other: `###### Jitsi Settings:
+* Embedded: |{{.Embedded}}|
+* Naming Scheme: |{{.NamingScheme}}|`,
```

Comment:
Instead of using string replacments I would prefer the use of normal string instead of raw one which are just contradicted. A translator doesn't know what `|` means in this context.

Reply:
If we do that, we need to make it in one single string to extract it properly with goi18n, what makes the string, very long and and to read with the `\n` charcters in the middle. This solution is, I think, good enough for the size of the translations that we have, the number of translations that we have to review here is not going to be a lot, so I think is not going to generate so much problems.

Reply:
The translations string in `1i8n/` already contain `\n` characters. Hence using a string would not change anything for them.

While I see your point for command help, these string here seam different because they are quite short.

Reply:
Exactly, edit and see this in the code is going to be a pain if we have to use multiple translations strings, or one big translation strings with `\n` instead of real new lines. I don't think the expectation of any translator is to have to translate symbols like `|`, and if any error happens during the translation, is going to be fixed quickly, as I said, I don't expect a lot of maintenance effort in the translations here.